### PR TITLE
Add a skybox & fixes for running from a non root directory

### PR DESCRIPTION
**Pull request by NyarukoDev**
When zooming out on the site, the plane is in a void, this adds a sky gradient.

Before:
image

After:
image

### CODE REVIEW

**Comment by hedgehog1029** [comment]
what

ok yea

File: assets/style.css [hunk 1]
```diff
@@ -27,7 +32,7 @@ body {
 .block>* {
 	position: absolute;
 	background-size: 128px;
-	background-image: url('/assets/stone_slab_side.png');
```
Comment (hedgehog1029) [inline assets/style.css]: these URLs were correct before - if you're having issues locally, run a http server rather than opening html files directly

could u revert these assets-based changes

File: index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
 		<meta property="og:description" content="Minecraft Double Smooth Stone Slab">
 		<meta property="og:type" content="website">
 		<meta property="og:image" content="https://minecraftdoublesmoothstoneslab.website/minecraftdoublesmoothstoneslab.jpg">
-		<link rel="stylesheet" type="text/css" href="/assets/style.css" />
+		<link rel="stylesheet" type="text/css" href="assets/style.css" />
```
Comment (hedgehog1029) [inline index.html]: also revert this change
```suggestion
		<link rel="stylesheet" type="text/css" href="/assets/style.css" />
```

File: assets/style.css (hunk 1 above)
Comment (NyarukoDev) [reply assets/style.css]: Reverted them, was mainly using them to make the site loadable from directories besides `/`.
